documentation/using_plugins.md

```diff
@@ -44,6 +44,37 @@ groups:
 
 This reference will be resolved as soon as Bolt runs.
 
+It is important to understand that fundamentally plugins are used to reference data. The simplest example of this concept is the [yaml](#yaml) plugin. With the yaml plugin, you can effectively "insert" yaml from one file into another. For example, this allows you to organize an inventory into multiple files.
```

Comment:
Unless YAML is between backticks (therefore a command), cap the entire thing: YAML.